libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3716,11 +3689,6 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     units: ethereumUnits("ETH", "ETH"),
     ethereumLikeInfo: {
       chainId: 42161,
```

Comment:
This is the latest element in currencies.ts ethereumLikeInfo :thinking: We could add it in coinconfig but this won't be updated and i'm not sure this makes sense... WDYT ?

I think long term a lot more will be pulled remotely soooo

Reply:
I don't think it's relevant to have the chainId as a "config", it's an attribute of a currency